docs/.vuepress/tools/snippet-transform/helpers/previewLogger.js

```diff
@@ -0,0 +1,106 @@
+const path = require('path');
+const fs = require('fs');
+
+const LOG_FILE_NAME = 'react-translation-log.html';
```

Comment:
Until we make a script to compare the log files, I'll add it to `.gitignore`, but it most probably will be removed from it later on.

Reply:
As agreed, this file is not ignored, because the requirements say it must be committed as a way of verifying the correctness of the transformation.

However, the file name is inconsistent. I think it should be `snippet-transformer-log-react.html`